packages/new-editor/src/components/editor/variables/detail/Variable.tsx

```diff
@@ -2,15 +2,20 @@ import { Fieldset, Input, SimpleSelect, Textarea, ToolbarTitle } from '@axonivy/
 import { metadataOptions, type Variable } from '../../data/Variable';
 
 type VariableProps = {
-  variable: Variable;
+  variable: Variable | undefined;
 };
 
 export const VariableDetail = ({ variable }: VariableProps) => {
+  const title = <ToolbarTitle>Variable Configuration</ToolbarTitle>;
```

Comment:
I think you shouldn't reuse this toolbartitle here. it may causes unwanted changes, if we ever change this component and don't have this usage here in mind...

Reply:
instead you could just e.g. leave a message about select a variable or something like that.